src/library/scala/collection/immutable/BitSet.scala

```diff
@@ -81,7 +81,8 @@ object BitSet extends BitSetFactory[BitSet] {
   }
 
   /** $bitsetCanBuildFrom */
-  implicit def canBuildFrom: CanBuildFrom[BitSet, Int, BitSet] = bitsetCanBuildFrom
+  implicit def canBuildFrom: CanBuildFrom[BitSet, Int, BitSet] = ReusableCBF
+  private[this] val ReusableCBF = bitsetCanBuildFrom
```

Comment:
Here's another case where we don't need a new `val`. All monomorphic ones can use `val canBuildFrom`.

Reply:
done